client/app/components/packages/scope-of-work-draft/attached-documents.hbs

```diff
@@ -38,7 +38,7 @@
   </ul>
 
   <Packages::Attachments
-    @package={{@model}}
+    @package={{@form.data}}
     @fileManager={{@model.fileManager}}
```

Comment:
What about the @model.fileManager? should it be updated to @form.data.fileManager?